.github/workflows/comment_bot.yml

```diff
@@ -26,8 +26,7 @@ on:
 
 permissions:
   contents: read
-  pull-requests: write
```

Comment:
Just checking, does the `issue_assign` job need this permission? I guess maybe not because it's modifying the issue and not the PR?

Reply:
I don't think it does since it has its own permissions scoped anyway:

https://github.com/jonkeane/arrow/blob/fa538b35c21e810b20e2e3bfc1b5e6dd8419cbf4/.github/workflows/comment_bot.yml#L59-L60

Reply:
But that is also an empirical question (easiest to test on main, I'm happy to follow on if it stops working)

Reply:
`take` is just fine: https://github.com/apache/arrow/issues/44654#issuecomment-2458248499

Reply:
It seems that this is needed for `@github-actions crossbow ...`: https://github.com/apache/arrow/pull/44658